codejudge/app/views/registrations/new.html.erb

```diff
@@ -65,6 +65,16 @@
     </div>
   <% end %>
 
+  <%= form.label :LoginType, "Login Type" %>
+  <%= form.text_field :password_confirmation %>
+  <select name="logintype">
+  <option value=""></option>
+  <option value="2">Instructor</option>
+  <option value="3">Student</option>
+</select>
+<% end %>
```

Comment:
Remove end from here code is breaking with that tag

Reply:
done